### PR TITLE
Change some lists to domains

### PR DESCRIPTION
**Pull request by GiddyGoatGaming**
Realized a 1hosts list was still in hosts format so I went through lists and found some domains variants to cut down on processing time. Also changed bongochong & someonewhocares to ipv4/ipv6 variants for greater coverage (even though I don’t use ipv6).

### CODE REVIEW

File: edit_here_to_add_blocklists.json [hunk 1]
```diff
@@ -147,7 +147,7 @@
     "loc": "custom_filters/normal/id12.txt",
     "name": "someonewhocares",
     "category": "Ads, Privacy",
-    "source": "https://someonewhocares.org/hosts/hosts",
+    "source": "https://someonewhocares.org/hosts/ipv6zero/hosts/",
```
Comment (bhanupratapys) [inline edit_here_to_add_blocklists.json:150]: No need of changing this, server by default blocks both A and AAAA records for the domain present in the list.

File: edit_here_to_add_blocklists.json [hunk 2]
```diff
@@ -1227,7 +1227,7 @@
     "loc": "custom_filters/normal/id102.txt",
     "name": "bongochong (CombinedPrivacy)",
     "category": "Ads, Privacy",
-    "source": "https://raw.githubusercontent.com/bongochong/CombinedPrivacyBlockLists/master/newhosts-final.hosts",
+    "source": "https://raw.githubusercontent.com/bongochong/CombinedPrivacyBlockLists/master/newhosts-final-Dual.hosts",
```
Comment (bhanupratapys) [inline edit_here_to_add_blocklists.json:1230]: Same here, revert this too.

File: edit_here_to_add_blocklists.json (hunk 1 above)
Comment (GiddyGoatGaming) [reply edit_here_to_add_blocklists.json:150]: > 

Ok il switch both lists right away

**Review by bhanupratapys — CHANGES_REQUESTED** [review]
Requested some changes.

**Review by bhanupratapys — APPROVED** [review]
Merging it!

**Comment by GiddyGoatGaming** [comment]
Yay thanks!

**Pull request merged**